docs/API.md

```diff
@@ -604,6 +604,17 @@ query HelloWorld {
 | http.status_code | The HTTP status code of the response.                     |
 | http.headers.*   | A recorded HTTP header.                                   |
 
+#### Knex
+
+##### Tags
+
+| Tag              | Description                                                |
+|------------------|------------------------------------------------------------|
+| component        | `knex`                                                     |
+| db.type          | The database driver being used such as sqlite3, etc        |
+| db.user          | The database user used by the driver to connect to the db. |
+| db.statement     | The SQL statement executed by the traced query.            |
```

Comment:
Would you recommend setting `db.instance` if reliably obtained?

Reply:
Will look.

Reply:
Done